app/controllers/api/v1/search_terms_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V1
+    class SearchTermsController < ApplicationController
+      def import
+        return render :json => { :errors => 'No file added' }, :status => 422 if params[:file].nil?
+        return render :json => { :errors => 'Only CSV files allowed' }, :status => 422 unless params[:file].content_type == 'text/csv'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/ffe479ef97670c7333f37452d55369f226bbc764/app/controllers/api/v1/search_terms_controller.rb#L6"></span>Line is too long. [134/120]</td>
    </tr>
  </tbody>
</table>